wandb/jupyter.py

```diff
@@ -238,7 +238,7 @@ def save_history(self):
                     "language_info": language_info,
                 },
             )
-            state_path = os.path.join("code", "_session_history.ipynb")
+            state_path = os.path.join(wandb.run.dir, "code", "_session_history.ipynb")
```

Comment:
We join `wandb.run.dir` with state_path on line 246 below.  We want the config to only contain the relative path to the code, this would break the frontend.  Have you confirmed it works?